libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/LoginActivity.java

```diff
@@ -76,21 +95,30 @@
  *
  * The bulk of the work for this is actually managed by OAuthWebviewHelper class.
  */
-public class LoginActivity extends AccountAuthenticatorActivity
-        implements OAuthWebviewHelperEvents {
+public class LoginActivity extends FragmentActivity
```

Comment:
`AccountAuthenticatorActivity` was deprecated and did not provide much functionality.  I have replicated everything it provided with the `accountAuthenticatorResult` and `mAccountAuthenticatorResponse` variables in this class.